code/modules/vore/fluffstuff/custom_items_vr.dm

```diff
@@ -907,12 +907,157 @@
 
 		rimplant.reagents.remove_any(rimplant.transfer_amount)
 
+//Draycu: Schae Yonra
+/obj/item/weapon/implant/reagent_generator/yonra
+	name = "egg laying implant"
+	desc = "This is an implant that allows the user to lay eggs."
+	generated_reagents = list("egg" = 2)
+	usable_volume = 500
+	transfer_amount = 50
+
+	empty_message = list("Your feathery lower belly feels smooth and empty. For now...", "The lack of clacking eggs in your abdomen lets you know you're free to continue your day as normal.",  "The reduced pressure in your lower belly tells you there are no more eggs.", "With a soft sigh, you can feel your lower body is empty.  You know it will only be a matter of time before another batch fills you up again, however.")
+	full_message = list("Your feathery lower belly looks swollen with irregular bumps, and feels very heavy.", "Your feathery covered lower abdomen feels really heavy, making it a bit hard to walk.", "The added weight from your collection of eggs constantly reminds you that you'll have to lay soon!", "The sounds of eggs clacking as you walk reminds you that you will have to lay soon!")
+	emote_descriptor = list("an egg right out of Yonra's feathery crotch!", "into Yonra's belly firmly, forcing her to lay an egg!", ", making Yonra gasp and softly moan while an egg slides out.")
+	var/verb_descriptor = list("squeezes", "pushes", "hugs")
+	var/self_verb_descriptor = list("squeeze", "push", "hug")
+	var/short_emote_descriptor = list("lays", "forces out", "pushes out")
+	self_emote_descriptor = list("lay", "force out", "push out")
+	random_emote = list("hisses softly with a blush on her face", "yelps in embarrassment", "grunts a little")
+	assigned_proc = /mob/living/carbon/human/proc/use_reagent_implant_yonra
+
+/obj/item/weapon/implant/reagent_generator/yonra/implanted(mob/living/carbon/source)
```

Comment:
Why do you need to override this? It doesn't look like it has any character-specific code in it.

Reply:
Not sure. Just how it's been done every other time, so I just followed the format.

This does
```c
processing_objects += src
+	to_chat(source, "<span class='notice'>You implant [source] with \the [src].</span>")
+	source.verbs |= assigned_proc
+	return 1
```
While reagent_generator does
```c
	processing_objects += src
	to_chat(source, "<span class='notice'>You implant [source] with \the [src].</span>")
	assigned_proc = new assigned_proc(source, verb_name, verb_desc)
	return 1
```